templates/keras_cv/index.md

```diff
@@ -88,25 +89,27 @@ to take effect.
 
 ```python
 import tensorflow as tf
-from tensorflow import keras
 import keras_cv
 import tensorflow_datasets as tfds
+import keras_core as keras
 
 # Create a preprocessing pipeline with augmentations
 BATCH_SIZE = 16
 NUM_CLASSES = 3
-augmenter = keras.Sequential(
+augmenter = keras_cv.layers.Augmenter(
```

Comment:
I couldn't find the `Augmenter` layer here: https://keras.io/api/keras_cv/layers/augmentation/ Do we need to add it, or am I just missing it?